tests/common.go

```diff
@@ -155,39 +165,58 @@ func ValidateContext(ctx *scheduler.Context) {
 			for _, vol := range vols {
 				Step(fmt.Sprintf("validate if %s app's volume: %v is setup", ctx.App.Key, vol), func() {
 					err := Inst().V.ValidateVolumeSetup(vol)
-					expect(err).NotTo(haveOccurred())
+					if vState == 0 {
+						expect(err).To(haveOccurred())
+					} else {
+						expect(err).NotTo(haveOccurred())
+					}
 				})
 			}
 		})
 	})
 }
 
 // ValidateVolumes is the ginkgo spec for validating volumes of a context
-func ValidateVolumes(ctx *scheduler.Context) {
+func ValidateVolumes(ctx *scheduler.Context, vState int) {
```

Comment:
Looks like we need two functions `ValidateVolumes()` functions instead of a single function with a `vState` flag.

Reply:
Changed the way of passing this flag , it is now sent as part of ScheduleOptions